src/vault.py

```diff
@@ -121,6 +121,65 @@ def configure_kv_mount(self, name: str):
                 path=name,
             )
 
+    def enable_pki_engine(self, path: str):
```

Comment:
I would suggest forcing keyword argument usage for all the methods added:
```suggestion
    def enable_pki_engine(self, *, path: str):
```